ddR/R/ddR.R

```diff
@@ -246,6 +255,8 @@ dmapply <- function(FUN ,..., MoreArgs=list(),
   # Ensure that ... arguments are of equal length. length() works correctly for data.frame,
   # arrays, and lists
   lens <- vapply(dargs,function(x) {
+    # TODO Clark: Why even allow the possibility of this happening where
```

Comment:
Does session mean R session? How would you support the following case: I start R,  run an algorithm on a small dataset using parallel, and then use Spark with ddR for a larger dataset. We should not force someone to restart the R session.